spec/controllers/followings_controller_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'spec_helper'
+
+describe FollowingsController do
+  describe "GET index" do
+    it_behaves_like "a security guard" do
```

Comment:
I agree that it doesn't read well in the code, but I named it so because it reads better when run the spec in documentation mode.
```
UsersController
  GET show
    behaves like a security guard
      redirects to the root page
```
I renamed it "a gatekeeper redirecting an unauthenticated user" :smiley:. What do you think?